### DISCUSSION TITLE
Paypal

### DISCUSSION THREAD

**Discussion by alucard2303** [hotspotbilling/phpnuxbill#7] [Ideas]
Hello, greetings from Mexico, I would like that you could also use the paypal payment gateway or a so-called payment market, it would be excellent, and I contributed to the translation of the system into Spanish, excellent API, I will always try to continue with the translation into Spanish to your system so that others can use it for your ease, thank you very much and regards

**Comment by ibnux** [comment]
Ok

Will add paypal payment gateway

If you enqble spanish translation, and some text still english, it will automatically add to spanish translation, so you can easily translate

**Reply by alucard2303** [reply]
thanks and greetings friend

**Comment by ibnux** [comment]
You can follow update in here  
[https://github.com/orgs/hotspotbilling/projects/1/views/1](https://github.com/orgs/hotspotbilling/projects/1/views/1)

Repository:  
[https://github.com/hotspotbilling/phpnuxbill-paypal](https://github.com/hotspotbilling/phpnuxbill-paypal)

**Comment by ibnux** [comment]
Done, please try it

i just try it on sandbox

**Comment by alucard2303** [comment]
Thanks I'll try it, cheers
